rust/main/agents/relayer/src/msg/metadata/base.rs

```diff
@@ -189,7 +189,7 @@ impl Deref for MessageMetadataBuilder {
 
 #[async_trait]
 impl MetadataBuilder for MessageMetadataBuilder {
-    #[instrument(err, skip(self), fields(destination_domain=self.destination_domain().name()))]
+    #[instrument(err, skip(self, message), fields(destination_domain=self.destination_domain().name()))]
```

Comment:
i assume we'd generally still have the message somewhere in the spans?

Reply:
we do yes, confirmed with e2e and RC logs that the lifecycle is still trackable via the message ID: https://cloudlogging.app.goo.gl/Koyyzv992JQh6cp8A